app/controllers/Emergency.scala

```diff
@@ -30,27 +30,16 @@ class Emergency(
     val reissueTopic = "Your login session has not been extended"
 
     (for {
-      publicKey <- loginPublicSettings.publicKey
```

Comment:
Note also that `publicSettings.publicKey` was previously optional (although that ended with  https://github.com/guardian/pan-domain-authentication/pull/151 in Panda v6), and `publicSettings.verification` (introduced with https://github.com/guardian/pan-domain-authentication/pull/150 in Panda v7) is **not optional**.